specifyweb/backend/stored_queries/tests/test_views/test_timestamp.py

```
@@ -0,0 +1,48 @@
+                "sorttype": 0,
+                "position": 1,
+                "isdisplay": True,
+                "operstart": 8,
+                "startvalue": "",
+                "isnot": False,
+                "isstrict": False,
+                "_tableName": "SpQueryField"
+            }
+        ]
+
+        response = c.post(f'/stored_query/ephemeral/', query, content_type="application/json")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)